src/ui_parts/viewport.gd

```diff
@@ -37,12 +37,18 @@ func clamp_view() -> void:
 
 
 # TODO implement the ability to zoom in a specific area.
-func zoom_in() -> void:
-	zoom_level *= sqrt(2)
+func zoom_in(pressed_button: bool) -> void:
+	if GlobalSettings.invert_zoom and not pressed_button:
+		zoom_level /= sqrt(2)
+	else:
+		zoom_level *= sqrt(2)
 	center_frame()
 
-func zoom_out() -> void:
-	zoom_level /= sqrt(2)
+func zoom_out(pressed_button: bool) -> void:
+	if GlobalSettings.invert_zoom and not pressed_button:
+		zoom_level *= sqrt(2)
+	else:
+		zoom_level /= sqrt(2)
```

Comment:
This is not where it should be done, instead, GodSVG should use the invert_zoom value to decide which one to call in the first place.

Reply:
so would this be better?
```gd
		match event.button_index:
			MOUSE_BUTTON_WHEEL_UP:
				if GlobalSettings.invert_zoom:
					zoom_out()
				else:
					zoom_in()
			MOUSE_BUTTON_WHEEL_DOWN:
				if GlobalSettings.invert_zoom:
					zoom_in()
				else:
					zoom_out()
```

Reply:
Yes.